R/errors.R

```diff
@@ -327,7 +327,10 @@ chain_end <- function(data, success_fun=success_continue,
 #' @param data A data frame
 #' @export
 clear_results <- function(data) {
-  attr(data, "assertr_results") <- NULL
-  attr(data, "assertr_errors") <- NULL
+  is_data_corrupted <- attr(data, "is_corrupted")
+  if (!isTRUE(is_data_corrupted)) {
```

Comment:
This is very strange if expression - isn't `is_data_corrupted` already a boolean? why don't we just write `!is_data_corrupted`. Otherwise, is there a function `isFALSE`? It is more readable than `!isTRUE`

Reply:
Not always data has attribute "is_corrupted", then is_data_corrupted is NULL so not boolean (and if crashes as result).
There is no isFALSE in base R. Please check actual solution, is it more readable?